libpod/container.go

```diff
@@ -671,7 +673,7 @@ func (c *Container) Start() error {
 // to stop the container, and if it has not stopped after the given timeout (in
 // seconds), uses SIGKILL to attempt to forcibly stop the container.
 // If timeout is 0, SIGKILL will be used immediately
-func (c *Container) Stop(timeout int64) error {
+func (c *Container) Stop(timeout uint) error {
```

Comment:
We need a better way of knowing whether to use or ignore the timeout parameter. Maybe maybe it a *uint and if it's nil we use StopTimeout, else we use the given timeout?

Reply:
I think we just move the handling to the podman/stop.go

Reply:
I still don't like this, because it ignores the StopTimeout we configured and forces you to manually set one outside of libpod. Really wish we had a good way of telling it to explicitly override the timeout. If Go allowed operator overloading I'd say we should have a Stop() and Stop(timout uint) with the first using the configured timeout and the second overriding it.

Reply:
I think we treat it as a required field for now.  No clean way to handle it otherwise, since we can't really know the users intention.